scripts/collection_from_items.py

```diff
@@ -6,19 +6,92 @@
 from boto3 import client
 from linz_logger import get_log
 
-from scripts.cli.cli_helper import coalesce_multi_single
+from scripts.cli.cli_helper import coalesce_multi_single, nullable_str, valid_date
 from scripts.files.fs_s3 import bucket_name_from_path, get_object_parallel_multithreading, list_json_in_uri
 from scripts.logging.time_helper import time_in_ms
 from scripts.stac.imagery.collection import ImageryCollection
+from scripts.stac.imagery.generate_metadata import ElevationSubtypes, ImagerySubtypes, generate_description, generate_title
 from scripts.stac.imagery.provider import Provider, ProviderRole
 
 
+# pylint: disable-msg=too-many-locals
 def main() -> None:
     parser = argparse.ArgumentParser()
+    parser.add_argument("--preset", dest="preset", required=True, help="Standardised file format. Example: webp")
     parser.add_argument("--uri", dest="uri", help="s3 path to items and collection.json write location", required=True)
     parser.add_argument("--collection-id", dest="collection_id", help="Collection ID", required=True)
-    parser.add_argument("--title", dest="title", help="Collection title", required=True)
-    parser.add_argument("--description", dest="description", help="Collection description", required=True)
+    parser.add_argument(
+        "--subtype",
+        dest="subtype",
+        help="Dataset subtype description",
+        required=True,
+        choices=[
+            ImagerySubtypes.AERIAL,
```

Comment:
You can use `list(SomeEnum)` to get all the entries in an enum.

Reply:
This didn't seem to work, I got the following errror:
```
collection_from_items.py: error: argument --subtype: invalid choice: 'Urban Aerial Photos' (choose from [<ImagerySubtypes.SATELLITE: 'Satellite Imagery'>, <ImagerySubtypes.UR
BAN: 'Urban Aerial Photos'>, <ImagerySubtypes.RURAL: 'Rural Aerial Photos'>, <ImagerySubtypes.AERIAL: 'Aerial Photos'>, <ImagerySubtypes.HISTORICAL: 'Scanned Aerial Photos'>]
, [<ElevationSubtypes.DEM: 'DEM'>, <ElevationSubtypes.DSM: 'DSM'>])
```

I changed it to: `choices=[type.value for type in ImagerySubtypes] + [type.value for type in ElevationSubtypes],`, this works, I'm uncertain if it is the best method for getting these values.